vm/runtime.ts

```diff
@@ -666,6 +666,9 @@ module J2ME {
     TWO_PWR_32_DBL = 4294967296,
     TWO_PWR_63_DBL = 9223372036854776000,
 
+    // The target amount of free memory before garbage collection is forced on unwind.
+    FREE_MEMORY_TARGET = 4086,
```

Comment:
We set *ASMJS_TOTAL_MEMORY* to 32MiB in the default Makefile, and we change its value to 8MiB in runwhatsapppreview.mk, so we should set *FREE_MEMORY_TARGET* in the Make files too, so we can set it relative to *ASMJS_TOTAL_MEMORY*.